code/index.html

```diff
@@ -1,30 +1,140 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>My business webiste</title>
+  <link rel='stylesheet' href='https://unpkg.com/emoji.css/dist/emoji.min.css'>
+  <link rel="stylesheet" href="style.css">
+  <link rel="shortcut icon" href="./Images/logoproject3.jpg">
+
+</head>
+
+<body>
+
+  <header class="navbar">
+    <ul class="navbar-content">
+      <img class="icon-main" src="./Images/logoproject3.jpg" alt="icon-main">
+      <li>Home</li>
+      <li>About us</li>
+      <li><a href="#accordion-container" >FAQ</a></li>
+    </ul>
+
+    <img class="icon-mobile" src="./Images/burgerbar.jpg" alt="icon">
+
+  </header>
+
+  <artical>
+    <div class="fullvideo-container"></div>
+    <div class="fullvideo-content"></div>
+
+    <div class="fullvideo-textbox">
+      <h1 Class="header-text">Help us save our oceans</h1>
+
+     <h2 Class="base-text"><a href="#signup-form">Sign up to donate</a></h2>
+
+      <h3 class="info-text">You can donate to these organizations</h3>
+
+      <ul class="info-logos">
+        <li><a href="https://www.coralgardeners.org/" target="_blank"><img src="./Images/Coral.png" alt="Coral-logo"></a></li>
+        <li><a href="https://4ocean.com/" target="_blank"><img
+              src="./Images/4O.png" alt="4ocean"></a></li>
+        <li><a href="https://www.onepercentfortheplanet.org/" target="_blank"><img src="./Images/Top1.jpg" alt="Plasticocean"></a></li>
+      </ul>
+
+    </div>
+    <div class="Fullvideo-overlay"></div>
```

Comment:
Here you start the class name with a big `F`, but in other places you write only with small letters. Same here goes with the syntax - choose one way you are writing the class names and follow that throughout the project!
```suggestion
    <div class="fullvideo-overlay"></div>
```